packages/components/form/__specs__/text_field.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import { useForm } from "react-hook-form";
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+
+import { act, render as baseRender, waitFor, UserRenderResult } from "../test_utils";
+import { Form, TextField } from "../src";
+
+describe("<TextField />", () => {
+  const translations = {
+    errors: {
+      required: "This field is required",
+    },
+    hints: {
+      test_form: {
+        my_text_field: "Optional helper text"
+      }
+    },
+    labels: {
+      my_text_field: "This is my text field",
+    },
+    placeholders: {
+      my_text_field: "Enter value",
```

Comment:
This probably needs to be:

```suggestion
      test_form: {
        my_text_field: "Enter value",
      }
```